client/src/components/EventCard.js

```diff
@@ -14,11 +14,28 @@ const Card = styled.div`
   height: 93px;
   border-radius: 12px;
   h2 {
-    color: var(--highlight-clr-category);
+    color: ${(props) => {
+      switch (props.category) {
+        case "wine":
+          return "var(--highlight-clr-category1)";
+        case "coffee & tea":
+          return "var(--highlight-clr-category2)";
+        case "beer":
+          return "var(--highlight-clr-category3)";
+        case "mixed drinks":
+          return "var(--highlight-clr-category4)";
+        case "soft drinks":
+          return "var(--highlight-clr-category5)";
+
+        default:
+          return "var(--font-color-white)";
+      }
+    }};
+
     grid-row: 3;
     grid-column: 1 / 5;
-    font-family: var(--font-archivoblack);
-    font-size: 12px;
+    font: 11px/9px var(--font-archivoblack);
```

Comment:
Below you are working with EM. Why not here?

Reply:
initially started with px, going to convert everything into rem